flatdata-rs/Cargo.toml

```diff
@@ -3,8 +3,8 @@ name = "flatdata"
 version = "0.2.1"
 authors = ["boxdot <d@zerovolt.org>"]
 license = "Apache-2.0"
-description = "Rust implementation of heremaps/flatdata"
-repository = "https://github.com/boxdot/flatdata-rs"
+description = "Rust implementation of flatdata"
+repository = "https://github.com/heremaps/flatdata"
```

Comment:
Does the sub-path need to be mentioned here?

Reply:
This is designed to perform a `git clone`, so the subpath is not part of this field.